frontend/plan/src/styles/App.css

```diff
@@ -491,7 +491,7 @@ ul {
 
 @media all and (max-width: 480px){
   .box{
-    min-height: 500px;
+    min-height: calc(100vh - 8em);
```

Comment:
Try to avoid using calc -- it's buggy and not well supported. For this pull request though we can keep it in since the min height requirement isn't very important.